pluma/stream/zeromq.py

```diff
@@ -152,8 +152,8 @@ def __init__(self, eventcode: int, **kw):
             eventcode,
             streamtype=StreamType.UNITY,
             filenames=[
-                'VRTransform/Position_Frame1.bin',
-                'VRTransform/Position_Frame2.bin'],
+                'Unity/Position_Frame1.bin',
+                'Unity/Position_Frame2.bin'],
```

Comment:
```suggestion
                'Unity/VRTransform_Frame1.bin',
                'Unity/VRTransform_Frame2.bin'],
```

I would name the files as `VRTransform` since they contain more information than just position.

Reply:
Solved in a1f97d8